### ISSUE TITLE
Problem with update to jsonwebtoken 6.2 dependency

### ISSUE THREAD

**Issue by joewood** [Azure/azure-mobile-apps-node#423]
The following is erroring out when logging in with the JavaScript client SDK - in jsonwebtoken/sign.js

```javascript
71   if (typeof payload.exp !== 'undefined' && typeof options.expiresIn !== 'undefined') {
72	    throw new Error('Bad "options.expiresIn" option the payload already has an "exp" property.');
73 }
```

The SDK seems to be setting both `exp` on **payload** and `expiresIn` on **options**.

**Comment by danderson00** [comment]
Meant to respond earlier, thanks for the report. We'll get a fix out this week.

**Comment by danderson00** [comment]
Resolved by https://github.com/Azure/azure-mobile-apps-node/commit/404cbc87ad8948eaf396dca308a893ed5b4a2dcb

**Comment by danderson00** [comment]
@joewood Meant to mention, this was included in 2.1.4 which we released yesterday.
